scripts/spark/start-spark.sh

```diff
@@ -15,5 +15,4 @@
 
 set -e
 
-cd $SPARK_HOME
-./sbin/start-connect-server.sh --jars $SPARK_HOME/jars/spark-connect_2.12-3.5.1.jar
+$SPARK_HOME/sbin/start-connect-server.sh --jars $SPARK_HOME/misc/spark-connect_2.12-3.5.1.jar
```

Comment:
@rui-mo 
nit: the internal shell linter has highlighted that the argument should be enclosed in quotes to prevent globbing and word splitting.  can you please update this whenever you get a chance. Thanks!

```
$SPARK_HOME/sbin/start-connect-server.sh --jars "$SPARK_HOME/misc/spark-connect_2.12-3.5.1.jar"
```

more info here: https://github.com/koalaman/shellcheck/wiki/SC2086

Reply:
@bikramSingh91 Thanks for the pointer. Filed https://github.com/facebookincubator/velox/pull/10381 to quote the variables.